blog/2025/pcsx2-2.6/index.mdx

```diff
@@ -0,0 +1,154 @@
+---
+draft: false
+title: PCSX2 2.6.0 is now out!
+tags:
+  - devblog
+date: 2025-12-31
+description: PCSX2 2.6 Release Post
+hide_table_of_contents: false
+titleImage: /img/2-0-banner.webp
+image: /img/2-0-banner.webp
+authors:
+  - kamfretoz
+  - govanify
+  - pandubz
+  - jordan
+---
+
+Nearly 6 months after PCSX2 2.4.0 release, we are now nearing the end of 2025.
+
+**Today we are excited to unveil PCSX2 2.6.0!** We're even more excited to showcase the things that we have been working on.
+
+<!--truncate-->
+
+## PCSX2 2.6.0
+
+Now that the dust has settled down from [our biggest single release to date](../../2024/pcsx2-2-release/index.mdx), we decided to step back a bit and give more attention to an aspect that are sometimes overlooked, and that is _Quality of Life Features and Accessibility_! Our contributor have been hard at work with further refining the interface foundation that have been laid out since 2.0.0 to make PCSX2 more accessible to everyone!
+
+<!--PR List-->
+
+## Major Spotlight
+
+<PCSX2PRLink authors="SternXD" prNums="12740,12944,13294">
+  ### Better Feature Parity Between Big Picture Mode and Qt Interface
+</PCSX2PRLink>
+
+SternXD has been on a mission to bring up a near-perfect parity in function across the Big Picture Mode and Qt Interface. Thanks to his work, you can now adjust Network and HDD Settings, Create a Memory Card as well as login to RetroAchievements directly from Big Picture Mode!
```

Comment:
on a mission to achieve feature parity between Big Picture Mode and the Qt interface